mmv1/products/apphub/Application.yaml

```diff
@@ -43,12 +43,30 @@ async:
     path: 'error'
     message: 'message'
 custom_code:
+  constants: 'templates/terraform/constants/apphub_application.go.tmpl'
+custom_diff:
+  - 'apphubApplicationCustomizeDiff'
 examples:
   - name: 'application_basic'
     config_path: 'templates/terraform/examples/apphub_application_basic.tf.tmpl'
     primary_resource_id: 'example'
     vars:
       application_id: 'example-application'
+      location: 'us-east1'
+      scope_type: 'REGIONAL'
+    test_vars_overrides:
+      'location': '"us-east1"'
+      'scope_type': '"REGIONAL"'
+  - name: 'application_global_basic'
+    config_path: 'templates/terraform/examples/apphub_application_basic.tf.tmpl'
```

Comment:
is config_path actually necessary here? This is usually just used in for private overrides. I'm assuming other examples of using this were just accidentally copied over from the private repo

Reply:
Have updated basic and full examples to remove config_path. But for *_global_basic , we were intending to use the same config as basic with different values.